README.rst

```diff
@@ -219,6 +219,24 @@ serious update from my side, maybe ignore that, until I get there.
 Review applies here very much. I think Tutorial parts might be factored
 out to separate documents.
 
+******************
+ How to translate
+******************
+
+.. code:: bash
+   
+   # Generate the .pot files
+   cd intl
+   make gettext
```

Comment:
the `make` is kind of windows only for most people, problem?

Reply:
seems we could make it an invoke target, not a lot in the makefile

Reply:
it's easy to install make on windows